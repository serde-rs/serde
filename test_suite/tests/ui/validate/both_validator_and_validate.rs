use serde_derive::Deserialize;

#[derive(validator::Validate, Deserialize)]
#[serde(validate = "validator::Validate::validate")]
#[serde(validator)]
struct ValidatorStruct {
    #[validate(email)]
    mail: String,
}

fn main() {}
