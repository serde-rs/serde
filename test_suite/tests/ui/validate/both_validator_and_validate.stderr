error: can not define both `validator` and `validate`
 --> tests/ui/validate/both_validator_and_validate.rs:4:20
  |
4 | #[serde(validate = "validator::Validate::validate")]
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
